lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -420,22 +420,27 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   $base_url = url(base_path(), array('absolute'=> TRUE, 'language' => 'en-global'));
   $campaign_path = url(current_path(), array('absolute' => TRUE));
   $country_prefix_for_share = (dosomething_global_get_current_prefix()) ? strtoupper(dosomething_global_get_current_prefix()) : 'global';
-  $share_image = $base_url . 'sites/default/files/images/problem-' . $campaign->nid . '-' . $country_prefix_for_share . '.png';
+  if (dosomething_campaign_feature_on($campaign, 'enable_voter_registration', TRUE)) {
+      $vars['share_image'] = dosomething_image_get_themed_image_url($campaign->variables['share_image_nid'], 'landscape');
```

Comment:
@angaither you can, but it's optional, so I thought we'd want to just stick with the original?